toDoList/src/index.js

```diff
@@ -1,72 +1,92 @@
+/* eslint-disable no-use-before-define */
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Finish project',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Clean the house',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Take Walk',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Hangout with friends',
-    completed: true,
-    index: 3,
-  },
-  {
-    description: 'Rest for 8 hours',
-    completed: false,
-    index: 4,
-  },
-];
+const addForm = document.querySelector('#add-form');
+const todoList = document.querySelector('#todo-list');
+const removeCompletedBtn = document.querySelector('#remove-completed');
 
-// Function to iterate over tasks array and create HTML list items
-function showTasks() {
-  const todoList = document.getElementById('listContainer');
-  const items = document.createElement('ul');
-  items.classList.add('items');
-  todoList.innerHTML = '';
-  const taskTitle = document.createElement('p');
-  taskTitle.innerText = "Today's To Do";
-  taskTitle.classList.add('taskTitle');
-  todoList.appendChild(taskTitle);
-  const addTask = document.createElement('input');
-  addTask.setAttribute('placeholder', 'Add Task');
-  addTask.classList.add('addTask');
-  todoList.appendChild(addTask);
+let tasks = [];
+
+// Function to add a new task to the array and render it
+const addTask = (taskDescription, completed = false) => {
+  tasks.push({ description: taskDescription, completed });
+  renderTasks();
+};
```

Comment:
- [ ] Based on project requirements, you were meant to create a new file for all new functionality stated. Kindly do so with all the other functions as stated in the project requirements in the snapshot below :

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/222011846-0454ba76-3129-4213-aa67-617358d16937.png)